### DISCUSSION TITLE
Unable to locate an Android NDK

### DISCUSSION THREAD

**Discussion by beppo-ivel** [appcelerator/titanium_mobile#13686] [Q&A] [answered]
Trying to register ndk for titanium. First I installed the ndk via sdkmanager:
```

$ sudo env "PATH=$PATH" sdkmanager --install "ndk;25.1.8937393" --sdk_root=/opt/android-ndk/ --verbose

$ ls -al /opt/android-ndk/ndk/25.1.8937393/
total 1456
...
-rwxr-xr-x  1 root root     73 Dec 20 06:31 ndk-build
...

```
I create env variable (I also tried the path /opt/android-ndk/ndk/25.1.8937393):
```
export ANDROID_NDK_ROOT=/opt/android-ndk/ndk
export PATH=$PATH:$ANDROID_NDK_ROOT
```
 Then I tried to tell titanium where the ndk is installed:
`$ titanium config android.ndkPath /opt/android-ndk/ndk/25.1.8937393
`

But `ti info` returns:

>  !  Unable to locate an Android NDK.
     Without the NDK, you will not be able to build native Android Titanium modules.
     If you have already downloaded and installed the Android NDK, you can tell Titanium where the
     Android NDK is located by running 'titanium config android.ndkPath /path/to/android-ndk',
     otherwise you can install it by running 'titanium setup android' or manually downloading from
     https://developer.android.com/ndk.

**Comment by m1ga** [comment, marked as answer]
You can ignore that error. Since last year the NDK is installed when its needed by the CLI (https://github.com/tidev/titanium_mobile/pull/12711). NDK is installed `side-by-side` by the SDK now.

**Reply by m1ga** [reply]
https://github.com/tidev/node-titanium-sdk/pull/629 error message will be removed

**Reply by beppo-ivel** [reply]
Can I edit the docs?

**Reply by m1ga** [reply]
of course. All the repos are open source. you just have to find the one you want to edit: https://github.com/tidev/
and check https://github.com/tidev/titanium_mobile#contributing on how to contribute.

The "normal" API docs are in https://github.com/tidev/titanium_mobile/tree/master/apidoc the other pages in the docs are in https://github.com/tidev/titanium-docs
And of course the other repos if its a module documentation and so on
